src/sardana/macroserver/scan/gscan.py

```diff
@@ -1961,8 +1962,17 @@ def scan_loop(self):
 class CAcquisition(object):
 
     def __init__(self):
-        self._thread_pool = ThreadPool(name="ValueBufferTH", Psize=1,
-                                       Qsize=100000)
+        # protect older versions of Taurus (without the worker_cls argument)
+        # remove it whenever we bump Taurus dependency
+        try:
+            self._thread_pool = ThreadPool(name="ValueBufferTH",
+                                           Psize=1,
+                                           Qsize=100000,
+                                           worker_cls=OmniWorker)
+        except TypeError:
+            self._thread_pool = ThreadPool(name="ValueBufferTH",
+                                           Psize=1,
+                                           Qsize=100000)
```

Comment:
I would prefer that sardana fails if we are not using a minimum taurus with the worker_cls feature.
If we hide it the users may have this bug and don't know about it

Reply:
Yes, I agree with you. To do that we still need to ask taurus and PyTango to bump the version numbers and then bump the requirement. I propose that we defer it until the release.

Reply:
Ok, I understand that it is complicated to have to wait for new taurus and PyTango versions.
Maybe we can put a warning saying that the user might be affected by the bug?

Reply:
I added warning logs in 6d2583d. Thanks!